lib/mockbuilder.py

```diff
@@ -30,6 +30,14 @@
 CONF = config.get_config().CONF
```

Comment:
Regarding commit msg, it took me a while to understand that "yum repository files" here do not refer to the yum repository metadata files, but only to the yum repository config file. I'd prefer "Create yum repository config file when building packages"